src/kOS/Communication/ProcessorManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using kOS.Module;
+using kOS.Safe.Compilation;
+using kOS.Safe.Persistence;
+using kOS.Safe.Utilities;
+
+namespace kOS.Communication
+{
+    public class ProcessorManager
```

Comment:
You're right, looks like I forgot to delete this whole package when creating a commit.